### PR TITLE
Modify README.md about QEMU

### PR DESCRIPTION
**Pull request by junmin-kim**
TizenRT/README.md : QEMU is not board
TizenRT/build/configs/qemu/README.md : Add more description about information

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/qemu/README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ QEMU is a generic and open source machine emulator and virtualizer.
 
 ## Information
 
-QEMU uses lm3s6963-ek machine.
+QEMU and TizenRT support the lm3s6965-ek board.
```
Comment (sunghan-chang) [inline build/configs/qemu/README.md]: Do you want to show it in same line?
If you want to show it in other line, you should add two spaces at end of line.

**Comment by junmin-kim** [comment]
Is there any good word to replace "Improve"?
Fix, update, add are not proper here

File: build/configs/qemu/README.md (hunk 1 above)
Comment (junmin-kim) [reply build/configs/qemu/README.md]: I fixed it

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/qemu/README.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ QEMU is a generic and open source machine emulator and virtualizer.
...
```
Comment (btheosam) [inline build/configs/qemu/README.md]: Tizen RT can be configured and built for lm3s6965-ek board which is guranteed to run ONLY on QEMU emulator.

File: build/configs/qemu/README.md [hunk 3]
```diff
@@ -4,7 +4,9 @@ QEMU is a generic and open source machine emulator and virtualizer.
 
 ## Information
```
Comment (btheosam) [inline build/configs/qemu/README.md]: Delete this information bloc here. First should be Environment Set-Up with QEMU installation.

Comment (btheosam) [inline build/configs/qemu/README.md]: Targets Supported:
a. lm3s6963-ek

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/qemu/README.md [hunk 4]
```diff
@@ -4,7 +4,13 @@ QEMU is a generic and open source machine emulator and virtualizer.
...
 
-QEMU uses lm3s6963-ek machine.
+Targets Supported:
+
+1. lm3s6963-ek  
+NOTE:  
+Tizen RT supports two configurations for lm3s6963-ek board;
+- 64KB memory requirment configuration which matches with real memory of lm3s6963-ek board
```
Comment (sunghan-chang) [inline build/configs/qemu/README.md]: 1. requirment -> requirement
2. 64KB memory -> 64KB RAM

File: build/configs/qemu/README.md [hunk 5]
```diff
@@ -4,7 +4,13 @@ QEMU is a generic and open source machine emulator and virtualizer.
 
 ## Information
 
-QEMU uses lm3s6963-ek machine.
+Targets Supported:
+
+1. lm3s6963-ek  
+NOTE:  
+Tizen RT supports two configurations for lm3s6963-ek board;
```
Comment (sunghan-chang) [inline build/configs/qemu/README.md]: two configurations? It looks there are two folders under build/configs/qemu.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**